vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1350,8 +1350,16 @@ EXPORT(int, sceGxmPrecomputedFragmentStateSetAllTextures, SceGxmPrecomputedFragm
     return 0;
 }
 
-EXPORT(int, sceGxmPrecomputedFragmentStateSetAllUniformBuffers) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceGxmPrecomputedFragmentStateSetAllUniformBuffers, SceGxmPrecomputedFragmentState *precomputedState, Ptr<const void> const *bufferDataArray) {
+    if (!precomputedState || !bufferDataArray) {
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
+    }
+
+    UniformBuffers &uniform_buffers = *precomputedState->uniform_buffers.get(host.mem);
```

Comment:
Don't derefence it right away, should do a null safe check

```suggestion
    UniformBuffers *uniform_buffers = precomputedState->uniform_buffers.get(host.mem);
    
    if (!uniform_buffers) {
        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
    }
```

Just safe in case corrupted state